am-lib/build.gradle

```diff
@@ -92,8 +92,12 @@ dependencies {
     compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
     compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.0.16.Final'
     compile group: 'org.aspectj', name: 'aspectjrt', version: '1.9.4'
+    compile group: 'io.swagger', name: 'swagger-annotations', version: '1.5.0'
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
+    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
```

Comment:
can you pls check if all the three dependencies are needed?

Reply:
springfox-swagger2 and springfox-swagger2-ui removed not needed.